packages/client/test/test-utils/utils.ts

```diff
@@ -662,3 +663,35 @@ export class Multimap<K, V> {
         return Array.from(this.values.keys())
     }
 }
+
+export const createPartitionedTestStream = async (): Promise<Stream> => {
+    const client = new StreamrClient({
+        ...clientOptions,
+        auth: {
+            privateKey: await fetchPrivateKeyWithGas()
+        }
+    })
+    const stream = await createTestStream(client, module, {
```

Comment:
Maybe the `module` should be a parameter to `createPartitionedTestStream() `so IDs reflect the test name